diff -Nru adonthell-0.3.3/src/game.cc adonthell-0.3.3.new/src/game.cc
--- adonthell-0.3.3/src/game.cc	Mon Sep 30 15:05:41 2002
+++ adonthell-0.3.3.new/src/game.cc	Sat Mar 22 16:26:40 2003
@@ -39,8 +39,14 @@
 {
     Global_data_dir = game_dir;
 #ifndef WIN32
-    User_data_dir = getenv ("HOME");
-    User_data_dir += "/.adonthell";
+    char *etc_dir = getenv("ETC_DIR");
+    if (etc_dir) {
+        User_data_dir = etc_dir;
+        User_data_dir += "/adonthell";
+    } else {
+        User_data_dir = getenv ("HOME");
+        User_data_dir += "/.adonthell";
+    }
 #else
     User_data_dir = Global_data_dir;
 #endif
diff -Nru adonthell-0.3.3/src/prefs.cc adonthell-0.3.3.new/src/prefs.cc
--- adonthell-0.3.3/src/prefs.cc	Mon Sep 30 15:05:41 2002
+++ adonthell-0.3.3.new/src/prefs.cc	Sat Mar 22 16:28:26 2003
@@ -62,7 +62,11 @@
 
     // set the path to the adonthellrc file:
 #ifndef WIN32
-    adonthellrc = string (getenv ("HOME")) + "/.adonthell";
+    char *etc_dir = getenv("ETC_DIR");
+    if (etc_dir)
+        adonthellrc = string (etc_dir) + "/adonthell";
+    else        
+        adonthellrc = string (getenv ("HOME")) + "/.adonthell";
 #else
     adonthellrc = string (".");
 #endif
